### PR TITLE
components created

### PR DESCRIPTION
**Pull request by YevheniiaBaidina**
Added main, aside, content component and create hw structure

### CODE REVIEW

**Review by iliatcymbal — COMMENTED** [review]
There are some errors during the building:
http://joxi.ru/EA4vq9DfwRjnzm

**Comment by YevheniiaBaidina** [comment]
asign component added

File: src/app-component.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDom from 'react-dom';
```
Comment (iliatcymbal) [inline src/app-component.js:2]: we do not need ReactDom in this place

File: src/main/content/numbers/Numbers.js [hunk 2]
```diff
@@ -3,7 +3,28 @@ import React from 'react';
 import './numbers.scss'
 
 export const Numbers = (props) => {
+    let arr = [];
+    for(let i = +props.from; i <= props.to; i++) {
+        arr.push(i);
+    }
+    
+    if(props.odd !== undefined) {
```
Comment (iliatcymbal) [inline src/main/content/numbers/Numbers.js]: space after if

File: src/main/content/numbers/Numbers.js [hunk 3]
```diff
@@ -3,7 +3,28 @@ import React from 'react';
 import './numbers.scss'
 
 export const Numbers = (props) => {
+    let arr = [];
+    for(let i = +props.from; i <= props.to; i++) {
```
Comment (iliatcymbal) [inline src/main/content/numbers/Numbers.js]: space after for

File: src/main/content/Content.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
+import { Greeting } from './greeting';
```
Comment (iliatcymbal) [inline src/main/content/Content.js]: where is in this branch file greeting.js?

**Comment by iliatcymbal** [comment]
[https://github.com/Shkyrmetoffa/janeShkyrmetoffa/blob/20-11/src/app.js](https://github.com/Shkyrmetoffa/janeShkyrmetoffa/blob/20-11/src/app.js)
There is no ReactDom imported, but you're using it here. Add please.

Fix it and merge to master

**Review by iliatcymbal — APPROVED** [review]

**Pull request merged**